tools/run-mypy

```diff
@@ -75,8 +104,11 @@ if args.quick:
     extra_args.append("--quick")
 
 # run mypy
-if python_files:
-    rc = subprocess.call([mypy_command] + extra_args + python_files)
-    sys.exit(rc)
-else:
-    print("There are no files to run mypy on.")
+for repo, python_files in repo_python_files.items():
+    print("Running mypy for `{}`.".format(repo))
+    if python_files:
+        rc = subprocess.call([mypy_command] + extra_args + python_files)
+        if rc:
+            sys.exit(rc)
+    else:
+        print("There are no files to run mypy on.")
```

Comment:
I think we want it to check all 3 packages even if there are errors in the first one; so we should stored a `failed` variable instead of existing immediately on the first error.

Reply:
Okay, I was not sure about that bit.